lib/uniara_virtual_parser/services/absences.rb

```diff
@@ -0,0 +1,28 @@
+module UniaraVirtualParser
+	module Services
+		module Absences
+		def absences(token)
+			response = Client.get_with_token('/alunos/consultas/faltas',token)
```

Comment:
Missing space after the comma.
`'/alunos/consultas/faltas',token`
`'/alunos/consultas/faltas', token`